samples/highcharts/blog/cognitive-impairment-no-labels/demo.js

```diff
@@ -0,0 +1,161 @@
+function padTo2Digits(num) {
+    return num.toString().padStart(2, "0");
```

Comment:
We prefer to use single quotes when writing JavaScript code. In some demos you use double quotes, in others you use single - it's good to be consistence. Not a big issue tho